cmd/create.go

```diff
@@ -71,27 +72,59 @@ func init() {
 	//flagApplyAnnotations(createCmd, "name", "__kubicorn_parse_list")
 	flagApplyAnnotations(createCmd, "profile", "__kubicorn_parse_profiles")
 
+	RootCmd.SetUsageTemplate(usageTemplate)
 	RootCmd.AddCommand(createCmd)
 }
 
 type profileFunc func(name string) *cluster.Cluster
 
-var alias = map[string]profileFunc{
-	"amazon":       profiles.NewSimpleAmazonCluster,
-	"aws":          profiles.NewSimpleAmazonCluster,
-	"do":           profiles.NewSimpleDigitalOceanCluster,
-	"digitalocean": profiles.NewSimpleDigitalOceanCluster,
-	"do-centos":    profiles.CentosDigitalOceanCluster,
-	"aws-centos":   profiles.CentosAmazonCluster,
+type profileMap struct {
+	profileFunc profileFunc
+	description string
+}
+
+var profileMapIndexed = map[string]profileMap{
```

Comment:
I fear this list can be endless, centOS on GCE, DO, AWS, Azure, Packet, IBM .. Ubuntu on GCE, DO, AWS, Packet, IBM. We should have some generic logic for this stuff, which is common and distributions / CP specific logic can keep growing.